Saltern Reshuttle & AI Added

WhiteLightningTwister opened the pull request:
Reworked alot of major departments such as command, cargo, security likewise added AI. There will likely be some inital errors i plan on fixing.

# Description

The final rework for saltern for a while, adding many new things likewise fixing up departments to be more useful to the people who occupy them. Likewise making the map more maints friendly. Many additonal empty rooms that can be renovated by people during the shift.

# Overview

<img width="5536" height="4544" alt="Saltern-0" src="https://github.com/user-attachments/assets/459d375e-745a-4a67-9c6e-a6222a40f8ad" />

# AI

<img width="1058" height="782" alt="NewAI" src="https://github.com/user-attachments/assets/a70af4a2-2f4a-4806-9f03-2089d60ceae1" />
<img width="530" height="829" alt="NewAIpath" src="https://github.com/user-attachments/assets/c28094ce-e2e8-4747-86ba-cc011773790b" />

# Logistics
<img width="1022" height="738" alt="NewCargo" src="https://github.com/user-attachments/assets/797ac96e-d4ed-4dd6-8ca3-b9b1a8dca289" />
<img width="670" height="544" alt="SalvageDock" src="https://github.com/user-attachments/assets/0c2da35a-8971-4197-8b71-103390ab3c34" />
<img width="613" height="428" alt="NewMail" src="https://github.com/user-attachments/assets/dd99735f-6b54-42ee-8304-9c360dcd0530" />

# Security

<img width="1121" height="775" alt="NewSecurity" src="https://github.com/user-attachments/assets/d8d1c096-a8f4-447f-8ce3-74114babb5db" />
<img width="622" height="423" alt="NewCorpsmen" src="https://github.com/user-attachments/assets/5903cd65-43e7-47dd-bc98-bd4a03a03976" />
<img width="730" height="717" alt="NewSecBarracks" src="https://github.com/user-attachments/assets/797fdb46-37d8-496c-bb51-8338608cde29" />

# Command
<img width="966" height="708" alt="NewCommand" src="https://github.com/user-attachments/assets/32469560-bba6-4b48-926e-d333631153d6" />
<img width="714" height="712" alt="NewHOP" src="https://github.com/user-attachments/assets/53c30241-c2ba-402b-a08b-575a94d2d45d" />

List of changes - NOT changelog
---

- add: Security Barracks
- add: AI Sat
- add: Robotics Console
- add: Holopads
- add: Service Breakroom
- add: Finally added EVAC pods, three new ones.
- tweak: Moved & reworked Cargo
- tweak: Reworked command
- tweak: Moved & reworked HOP
- tweak: Reworked Security
- tweak: Reworked detective & armory
- tweak: Moved Telecoms
- tweak: Reworked Engineering PA & Containment area
- tweak: Made the pipes to the TEG to outside more visible & accessible.
- tweak: Converted former rooms into maints areas
- fix: Fixed atmos in botany


Changelog
---
:cl:
- Tweak: Saltern has received a rework, featuring a brand new AI sattelite, holopads, breakrooms, as well as changes to other departments.

WhiteLightningTwister committed:
Saltern Reshuttle & AI added

Reworked cargo, command, security likewise HOP. Including added AI. Holopads are all over the station to!

Mnemotechnician commented:
Please update your changelog to reflect the changes made by this PR. This is what will be shown in the changelog channel and in the in-game changelog menu.

Edited: I updated your PR description myself. Moved the changes to the map above the changelog and only left a short description of the overall changes. Feel free to adjust.

WhiteLightningTwister committed:
Salternreshuffle Fix

Fixed a minor bug with the map which was causing a build fail.

WhiteLightningTwister committed:
SalternReshuttle Fixed job mistake

Fixing a mistake where i placed the wrong job spawn on the map for the mail room.

southbridge-fur commented:
Review time

WhiteLightningTwister committed:
Finalized map File

Includes all the additional changes requested by south bridge.

Mnemotechnician commented:
@WhiteLightningTwister if you want AI to actually work, you will need to edit https://github.com/Floof-Station/Floof-Station/blob/067423ceebcaaad96f6830dc6794e62a47601fe9/Resources/Prototypes/Roles/Jobs/departments.yml#L128 on your branch and uncomment the inclusion of StationAi in the silicon department.
